zmsentities/src/Zmsentities/Collection/RequestList.php

```
@@ -46,7 +46,7 @@ public function toSortedByGroup(): array
      *
      * @return RequestList
      */
-    public function withCountList($countList)
+    public function withCountList(mixed $countList)
```

PHPStan (2.2.8)

[error] PHPDoc tag `@param` references unknown parameter: $countlist


(parameter.notFound)